src/Microsoft.ML.OnnxTransformer/OnnxTransform.cs

```diff
@@ -687,6 +723,422 @@ public NamedOnnxValue GetNamedOnnxValue()
                     return OnnxUtils.CreateNamedOnnxValue(_colName, _vBufferDense.GetValues(), _tensorShape);
                 }
             }
+
+            /// <summary>
+            /// <see cref="OnnxTransformer.GetDropColumnsNames"/>
+            /// </summary>
+            public string[] GetDropColumnsNames()
+            {
+                return _parent.GetDropColumnsNames();
+            }
+        }
+
+        /// <summary>
+        /// Similar to <see cref="ColumnBindings"/>, but this class will enable dropping columns from the input
+        /// schema, in order to let OnnxTransformer support the <see cref="ColumnSelectingTransformer"/> onnx export.
+        /// </summary>
+        [BestFriend] // MYTODO: Is this necessary?
+        internal sealed class Bindings // MYTODO: Should I move this inside OnnxDataTransform?
+        {
+            // MYTODO: Should I simply inherit from ColumnBindings, since everything is the same except for the constructor (specifically, only, the way it created the _colMap)?
```

Comment:
So the only difference between this and `ColumnBindings` is the code in the constructor (specifically only the code that builds _colMap and the output schema).

So I thought that inheriting from ColumnBindings might be an option, but it's sealed and also it seems it's meant to only add columns not remove them. So I've been hesitant to change that. Is it ok to simply copy all that code in here? Or would it be preferable to unseal it and inherit that in here?